### DISCUSSION TITLE
The bin directory has been deleted?

### DISCUSSION THREAD

**Discussion by lprimak** [philippe44/AirConnect#494] [Q&A] [answered]
Hi,

Looks like the bin directory has been deleted with no replacement.
What is the new way of downloading the binaries?
Thank you!

**Comment by philippe44** [comment, marked as answer]
Yes it has been. Instead, you have the AirConnect-X.Y.Z.zip in the root folder or you can go to release artifacts and download it from there. It was not the right place to have compiled versions and that was making my updates noisy.

**Reply by lprimak** [reply]
oh, I see.

README needs to be changed, the correct wget command would be `wget https://raw.githubusercontent.com/philippe44/AirConnect/master/AirConnect-<X.Y.Z>.zip` (note the case)

Thank you

**Comment by lprimak** [comment]
To be honest, I am not sure it's an improvement. Im forced to download a large file with all the executables, instead of a one small one.

**Comment by philippe44** [comment]
yes, but that's important to me...
